src/constants/WalletAndCurrencyConstants.ts

```diff
@@ -381,6 +381,11 @@ export const SPECIAL_CURRENCY_INFO: {
       privateKeyLabel: s.strings.create_wallet_import_input_key_or_seed_prompt,
       privateKeyInstructions: s.strings.create_wallet_import_input_key_or_seed_instructions
     },
+    uniqueIdentifierInfo: {
+      addButtonText: s.strings.unique_identifier_dropdown_option_note,
+      identifierName: s.strings.unique_identifier_note,
+      keyboardType: 'default'
+    },
```

Comment:
I see how this is needed, since different coins call the memo different things, but I would have assumed `EdgeCurrencyInfo.memoType` would play into this more, making the keyboard type unnecessary. I know Jon spent a bunch of time working on this; it might be worth talking to him to see if this can be removed.

Reply:
Only the maxLength param is used in the gui currently.